### PR TITLE
Organize global variable definitions

### PR DESCRIPTION
**Pull request by k4m4**
The code:
```
global x
global y
```
can be expressed as:
```
global x, y
```
---
Also:
Why Gif
(Please use one line for library imports...)

### CODE REVIEW

File: lanGhost.py [hunk 1]
```diff
@@ -7,24 +7,10 @@
     import logging
     logging.getLogger("scapy.runtime").setLevel(logging.ERROR)  # Shut up scapy!
 
+    import urllib.request, urllib.parse, netifaces, traceback, threading, telegram, requests, sqlite3, base64, socket, time, nmap, json, sys, os
```
Comment (xdavidhu) [inline lanGhost.py]: Thanks Kama, I will merge the first commit but I like the way the imports look so please revert that back to the original.

**Review by xdavidhu — CHANGES_REQUESTED** [review]

**Comment by k4m4** [comment]
@xdavidhu Changes made, as requested.

File: lanGhost.py [hunk 2]
```diff
@@ -34,11 +35,7 @@
 
 def refreshNetworkInfo():
     try:
-        global iface_mac
-        global ip_range
-        global gw_ip
-        global gw_mac
-        global ip
+        global iface_mac, p_range, w_ip, w_mac, p
```
Comment (xdavidhu) [inline lanGhost.py]: First letters are missing. Could you fix it?
Also please use the "🎨' emoji at the beginning of the commit.
Thanks Kama, but this pull request is getting way too complicated for what it is :D

**Review by xdavidhu — CHANGES_REQUESTED** [review]

File: lanGhost.py (hunk 2 above)
Comment (k4m4) [reply lanGhost.py]: Woops.. (facepalm)

**Comment by xdavidhu** [comment]
Thanks!

**Pull request merged**